### PR TITLE
Changed shebangs to python3

### PR DESCRIPTION
**Pull request by amaarquadri**
removed shebangs from files that are not executable

### CODE REVIEW

**Comment by amaarquadri** [comment]
This PR is aimed at shifting development to Ubuntu 20.04.1, upgrading to ROS Noetic, and upgrading to Python 3. This will be accompanied by an updated virtual machine for the entire team to use.
Perhaps this should be moved to a fork of the repository to allow others to continue development.

File: aquadrone_description/launch/upload_sub.launch [hunk 1]
```diff
@@ -2,5 +2,5 @@
     <arg name="model_file" default="v2.urdf.xacro"/>
     <arg name="param_name" default="robot_description"/>
 
-    <param name="$(arg param_name)" command="$(find xacro)/xacro.py '$(find aquadrone_description)/models/$(arg model_file)'"/>
```
Comment (amaarquadri) [inline aquadrone_description/launch/upload_sub.launch:5]: Explicitly adding .py is not necessary and is deprecated

File: aquadrone_sim_worlds/worlds/empty_underwater.world [hunk 2]
```diff
@@ -32,9 +32,10 @@
     </light>
 
     <physics name="default_physics" default="true" type="ode">
-      <max_step_size>$(arg max_step_size)</max_step_size>
+      <!--TODO: Get these parameters to work-->
```
Comment (amaarquadri) [inline aquadrone_sim_worlds/worlds/empty_underwater.world:35]: These parameters had always caused an warning on the command line and were not properly processed. Previously the warning was ignored and the program would continue, but now it causes a crash. Commenting out for now

File: path_planning/src/path_planning/state_machines/branching_state_machine.py [hunk 3]
```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```
Comment (amaarquadri) [inline path_planning/src/path_planning/state_machines/branching_state_machine.py:1]: shebang not needed in files that are not executable (i.e. not nodes)

File: pip_requirements.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
 simple-pid
 mock
-numpy==1.13
```
Comment (amaarquadri) [inline pip_requirements.txt:3]: update to most recent version

File: vision/src/faker.py [hunk 5]
```diff
@@ -8,85 +8,82 @@
 from aquadrone_msgs.msg import Vision_Array, Vision
 
 
+class Omniscient_Vision:
```
Comment (amaarquadri) [inline vision/src/faker.py:11]: This file had a mix of tabs and spaces which was fine in python 2, but crashes python 3. There should be no code changes here, only whitespace edits

**Review by waterloo-aquadrone-vm-account — APPROVED** [review]

**Pull request merged**